SPEC.md

```diff
@@ -559,6 +561,18 @@ Defined Kinds:
 * `ImageManifest`
 * `PodManifest`
 
+
+### AC Version Type
+
+The App Container specification aims to follow semantic versioning and retain forward and backwards compatibility within major versions.
+For example, if an implementation is compliant against version 1.0.1 of the spec, it shall remain compatible with the complete 1.x series.
```

Comment:
@philips I personally like forward-compatible models (e.g. golang), because forward and backward compatible is a tough contract to hold up. 

If the clarification needed is just `s/it shall remain/it is/` then that seems fine.